tools/prlint/lint.ts

```diff
@@ -0,0 +1,143 @@
+import * as path from 'path';
```

Comment:
This is not a new file. It's a copy of `lint.js` with type bindings added and 2-space indent.